tests/rgen/Microsoft.Macios.Generator.Tests/ConfigurationXUnit.cs

```diff
@@ -0,0 +1,11 @@
+using System.Reflection;
```

Comment:
I think this file should be in tests/common/ConfigurationXUnit.cs, that way it's usable if other test suites start using XUnit.